TrackingPerformance/Condor/condorJobs_reco.py

```diff
@@ -1,167 +1,223 @@
+        config.detector_model_list,
+    )
+
+    NEED_TO_CREATE_SCRIPTS = False
+
+    for theta, momentum, part, dect in iter_of_combined_variables:
+        for task_index in range(n_jobs_per_para_set):
+
+            output_file_name_parts = [
+                f"REC_{dect}",
+                f"{part}",
+                f"{theta}_deg",
+                f"{momentum}_GeV",
+                f"{config.N_EVTS_PER_JOB}_evts",
+                f"{task_index}",
+            ]
+            output_file_name = "_".join(output_file_name_parts)
+
+            input_file_name_parts = [
+                f"SIM_{dect}",
+                f"{part}",
+                f"{theta}_deg",
+                f"{momentum}_GeV",
+                f"{config.N_EVTS_PER_JOB}_evts",
+                f"{task_index}",
+            ]
+            input_file_path = Path("_".join(input_file_name_parts)).with_suffix(
+                ".edm4hep.root"
+            )
+            input_file = (
+                sim_eos_dir / part / input_file_path
+            )  # FIXME: reasonable that part is twice in the path?
+
+            # Check if the input file exists
+            if not input_file.exists():
+                print(f"Error: Input file {input_file} does not exist. Skipping job.")
+                continue
+            # Check if the output file already exists and has correct Nb of events
+            output_dir = rec_eos_dir / part
+            output_dir.mkdir(parents=True, exist_ok=True)
+            output_file = (output_dir / output_file_name).with_suffix("_edm4hep.root")
+
+            # FIXME: Issue #4
+            if CHECK_OUTPUT and output_file.exists():
+                root_file = ROOT.TFile(fspath(output_file), "READ")
+                events_tree = root_file.Get("events")
+                if events_tree and events_tree.GetEntries() == config.N_EVTS_PER_JOB:
+                    root_file.Close()
+                    continue
+                root_file.Close()
+            NEED_TO_CREATE_SCRIPTS = True
+
+            # Create aida output Dir
+            output_dir_aida = output_dir / "aida_outputs"
+            output_dir_aida.mkdir(exist_ok=True)
+
+            arguments = (
+                f" --GeoSvc.detectors=$K4GEO/FCCee/CLD/compact/{config.detector_model_list[0]}/{config.detector_model_list[0]}.xml"
+                + " --inputFiles "
+                + input_file
+                + " --outputBasename  "
+                + output_file_name
+                + f" --VXDDigitiserResUV={ResVDX_UV_[0]}"
+                + " --trackingOnly"
+                + " -n "
+                + config.N_EVTS_PER_JOB
+            )
+            command = f"k4run {config.rec_steering_file} " + arguments + " > /dev/null"
+
+            # Write bash script for job execution
+            bash_script = (
+                "#!/bin/bash \n"
+                f"source {config.setup} \n"
+                "git clone https://github.com/gaswk/CLDConfig.git \n"  # FIXME: see issues
```

Comment:
Please use https://github.com/key4hep/CLDConfig.git and I think `ResVDX_UV_` argument should be removed, at least for now